esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/flow.cylc

```diff
@@ -3,21 +3,27 @@
     UTC mode = True
 
 [task parameters]
-    metric = radiation_budget
+    fast = radiation_budget
+    medium = ensclus, heatwaves_coldwaves
 
 [scheduling]
+   cycling mode = integer
+   initial cycle point = 1
+   final cycle point = 5
     [[graph]]
-        R1 = install_cold => configure => process<metric> => compare<metric>
+        R1 = install_cold => configure => process<fast> & process<medium> => compare<fast> & compare<medium>
+        3/P2 = compare<fast>[-P2] => configure => process<fast> => compare<fast>
+        4/P3 = compare<medium>[-P3] => configure => process<medium> => compare<medium>
 
 [runtime]
     [[root]]
         script = rose task-run
         env-script = "eval $(rose task-env)"
         [[[environment]]]
             MODULE_NAME = {{ MODULE_NAME }}
+            OUTPUT_DIR = ${CYLC_WORKFLOW_SHARE_DIR}/cycle/${CYLC_TASK_CYCLE_POINT}
             USER_CONFIG_DIR = ${CYLC_WORKFLOW_RUN_DIR}/etc
             USER_CONFIG_PATH = ${USER_CONFIG_DIR}/config-user.yml
-            OUTPUT_DIR = ${CYLC_WORKFLOW_SHARE_DIR}/cycle/${CYLC_TASK_CYCLE_POINT}
```

Comment:
No functional change has been made here; would it be possible to revert this change, please?

Reply:
f3a0b8d823c6f787ff0ef952e0472277327498bb